nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -323,6 +323,15 @@ in {
       '';
     };
 
+    extraPackages = mkOption {
```

Comment:
For the sake of consistency, shouldn't these be added to nextcloud-cron & nextcloud-update-db?
Especially the former may also need things that one would add here.